Guide/jobs.markdown

```diff
@@ -91,6 +91,24 @@ instance Job EmailCustomersJob where
     maxAttempts = 3
 ```
 
+#### Backoff Strategy
+
+By default, retries will occur every 30 seconds, until maxAttempts is reached. To change the timing so that retries happen every 5 seconds, you can set:
```

Comment:
```suggestion
By default, retries will occur every 30 seconds, until `maxAttempts` is reached. To change the timing so that retries happen every 5 seconds, you can set:
```